files/__addonLocation__/rollup.config.mjs

```diff
@@ -7,6 +8,9 @@ const addon = new Addon({
   destDir: 'dist',
 });
 
+// Add extensions here, such as ts, gjs, etc that you may import
+const extensions = ['.js',<% if (typescript) { %> '.ts',<% } %> '.hbs'];
```

Comment:
**bugfix**

Adding `'.hbs'` caused my docs app to fail, possibly for `<template>`-tag components. (Tests failed as well.)

```sh
Uncaught (in promise) Error: Attempted to load a component, but there wasn't a component manager associated with the definition. The definition was: factory

at getInternalComponentManager (http://localhost:60952/assets/vendor.js:27620:15)
at ConstantsImpl.component (http://localhost:60952/assets/vendor.js:30878:64)
at resolveComponent (http://localhost:60952/assets/vendor.js:28297:22)
at encodeOp (http://localhost:60952/assets/vendor.js:30236:18)
at pushOp (http://localhost:60952/assets/vendor.js:30177:7)
at http://localhost:60952/assets/vendor.js:29825:7
at Compilers.compile (http://localhost:60952/assets/vendor.js:28527:7)
at compileStatements (http://localhost:60952/assets/vendor.js:30180:17)
at maybeCompile (http://localhost:60952/assets/vendor.js:30159:18)
at CompilableTemplateImpl.compile (http://localhost:60952/assets/vendor.js:30142:14)
```

```suggestion
const extensions = ['.js',<% if (typescript) { %> '.ts'<% } %>];
```

Reply:
Ah good catch! Thank you!